python/sample-apps/template.yml

```diff
@@ -16,15 +16,22 @@ Resources:
       CodeUri: ./function
       Description: Build OTel Python Lambda Sample from scratch
       MemorySize: 512
-      Timeout: 15
+      Timeout: 30
       Policies:
         - AWSLambdaBasicExecutionRole
         - AWSLambda_ReadOnlyAccess
         - AWSXrayWriteOnlyAccess
         - AmazonS3FullAccess
+        - Statement:
+          - Sid: AMPRemoteWrite
+            Effect: Allow
+            Action:
+            - aps:RemoteWrite
+            Resource: '*'
       Environment:
         Variables:
           AWS_LAMBDA_EXEC_WRAPPER: /opt/python/otel-instrument
+          PROTOCOL_BUFFERS_PYTHON_IMPLEMENTATION: python  # needed because https://github.com/open-telemetry/opentelemetry-python/issues/2717 / when new release of library is released this should be solved and can be removed 
```

Comment:
This issue is now resolved as the `opentelemetry-python` has pinned the protobuf version.